src/vignette/storage/static_assets.clj

```diff
@@ -0,0 +1,48 @@
+(ns vignette.storage.static-assets
+  (:require [vignette.storage.protocols :refer :all]
+            [org.httpkit.client :as http]
```

Comment:
Yeah its basically a GET but there is some S3 specific signing mojo included if you supply credentials. Without that there would be problems accessing non public Ceph/S3 files which is a problem guys implementing Mobile Config service encountered. Where they implemented access using simple HTTP GET.

But yeah once I finish the work required for Static Assets I plan to make some simple benchmarks for S3 code to see if the increase in throughput is worth the investment.